nupic/research/frameworks/pytorch/imagenet/imagenet_tune.py

```diff
@@ -294,6 +294,10 @@ def run(config):
     if "sigopt_config" in config:
         kwargs = dict(zip(kwargs_names, [SigOptImagenetTrainable,
                                          *tune.run.__defaults__]))
+    elif "imagenet_trainable" in config:
+        imagenet_trainable = config["imagenet_trainable"]
+        assert ImagenetTrainable in imagenet_trainable.mro()
```

Comment:
Why not `isinstance(imagenet_trainable , ImagenetTrainable )` ?

Reply:
Because it's a class definition, not a class.

Reply:
Got it. So How about [issubclass](https://docs.python.org/3/library/functions.html#issubclass) instead of accessing the `mro` for clarity?

Reply:
True, Didn't think of that. I thought it'd have the same problem, but I didn't test it. I'll make the change.

Reply:
See 9bfe916